webpack.test.config.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+var webpack = require('webpack'),
+    config = require('./webpack.common.config.js');
+
+// Only use the test entry file
+config.entry = {
+    'js/testRunner': __dirname + '/assets/tests/testRunner.js'
+};
+// Put a dummy value in place of the django reverse module in test mode
+config.externals = {
+    reverse: "undefined"
```

Comment:
Is it important that this is the string `"undefined"` and not the literal `undefined`?